mne/coreg.py

```diff
@@ -1959,9 +1962,20 @@ def compute_dig_mri_distances(self):
 
     @property
     def trans(self):
-        """Return the head-mri transform."""
+        """The head->mri :class:`~mne.transforms.Transform`."""
         return Transform('head', 'mri', self._head_mri_t)
 
+    @property
+    def fiducials(self):
```

Comment:
for me it's always a bit dangerous to have a property that does more than a getattr. It can create quiet silent slow downs when users access attributes in a loop. Maybe call it get_fiducials() to be explicit that it does stuff?

Reply:
I considered that but we already have `.trans` for getting the transform…
I also considered reworking the internals to only use the `DigMontage` everywhere, but that would've been a larger refactor…

Reply:
I can run some timing tests to see how long it takes here

Reply:
… I've now tried to transition away from `._lpa`, `._rpa`, `._nasion` to ` `DigMontage` that's exposed directly as `.fiducials`. Let's see if CI comes back green